torchaudio/models/rnnt.py

```diff
@@ -377,12 +377,17 @@ class _Joiner(torch.nn.Module):
     Args:
         input_dim (int): source and target input dimension.
         output_dim (int): output dimension.
+        use_tanh (bool, optional): if ``True``, uses Tanh rather ReLU activation. (Default: ``False``)
+
     """
 
-    def __init__(self, input_dim: int, output_dim: int) -> None:
+    def __init__(self, input_dim: int, output_dim: int, use_tanh: bool = False) -> None:
```

Comment:
instead of bool option `use_tanh`, how about we do a str option `activation` that can take on a value of either `"relu"` or `"tanh"`? this way, if we find later that other activation functions are viable, we can continue adding them without needing to update the option. 

example: https://github.com/pytorch/audio/blob/66185e0074d06330dff4ea84cbface3b83312ec4/torchaudio/models/emformer.py#L343